tests/test_tasks.py

```diff
@@ -22,4 +26,22 @@ def test_render_spider():
     assert test_file_content == rendered_content
 
 
+def test_gen_html():
```

Comment:
Generally speaking, if you can split out your assertions into multiple tests, then [that's probably a good idea](https://softwareengineering.stackexchange.com/a/7829). This feels like a place where you could do that.